core/domain/summary_services.py

```diff
@@ -413,3 +415,9 @@ def get_recently_published_exploration_summary_dicts():
         reverse=True)
 
     return _get_displayable_exp_summary_dicts(summaries)
+
+def get_displayable_exp_summaries(exploration_summaries):
```

Comment:
get_displayable_exp_summary_dicts

But actually I think it's fine to just make the other function public.